_docker/drupal/drupal-filesystem/web/themes/custom/rhdp/rhdp.theme

```diff
@@ -197,10 +197,16 @@ function rhdp_preprocess_page(&$variables) {
 function rhdp_preprocess_node(&$variables) {
   $variables['node'] = $variables['elements']['#node'];
   $node = $variables['node'];
-  $nid = $node->id();
+  if ($node->getType() == 'author') {
+    $variables['author_name'] = $node->getTitle();
+  }
+
   if ($node->getType() == 'events') {
     $variables['event_title'] = $node->getTitle();
       $variables['event_description'] = $variables['content']['field_description'][0]['#text'];
+      $toot = $node->get('field_more_details')->getValue()[0]['uri'];
+      $node->hasField('field_more_details') ? $variables['event_url']=$node->get('field_more_details')->getValue()[0]['uri'] : '';
```

Comment:
This line is a bit confusing.

basically:
if field 'field_more_details' exists set 'event_url' variable to a uri, otherwise empty string (not setting 'event_url') ?

Reply:
Yeah, I figured the ternary operator remove more nested if statements.